src/api/schema/types/ProjectUser.graphql

```diff
@@ -0,0 +1,19 @@
+type ProjectUser {
+  id: ID!
+  userId: Int!
+  projectId: Int!
+  role: String!
+}
+
+type CreateProjectUserResponse {
+  projectUser: ProjectUser
+  errors: [ValidationError]!
+}
+
+extend type Mutation {
+  createProjectUser(
+    userId: Int!
+    projectId: Int!
+    role: String!
```

Comment:
We could use a `GraphQLEnum` to validate the `role` (OWNER, CONTRIBUTOR to start with)